Algorithm.Python/BasicTemplateFuturesHistoryAlgorithm.py

```diff
@@ -36,8 +36,8 @@
 class BasicTemplateFuturesHistoryAlgorithm(QCAlgorithm):
 
     def Initialize(self):
-        self.SetStartDate(2013, 10, 8)
-        self.SetEndDate(2013, 10, 9)
+        self.SetStartDate(2016, 8, 17)
+        self.SetEndDate(2016, 8, 20)
```

Comment:
https://github.com/QuantConnect/Lean/tree/master/Data/future/usa/minute/es
We don't have this date range in the open source so it'll break the regression tests.